README.md

```diff
@@ -38,20 +65,143 @@ juju ssh kafka/leader 'kafka.topics --bootstrap-server $BOOTSTRAP_SERVERS --list
 
 Available Kafka bin commands can be found with:
 ```
-snap info kafka --channel rock/edge
+snap info charmed-kafka --channel latest/edge
 ```
 
-### Checklist
-- [x] Super-user creation
-- [x] Inter-broker auth
-- [x] Horizontally scale brokers
-- [x] Username/Password creation for related applications
-- [x] Automatic topic creation with associated user ACLs
-- [x] Persistent storage support with [Juju Storage](https://juju.is/docs/olm/defining-and-using-persistent-storage)
-- [x] TLS/SSL encrypted connections
-- [ ]  mTLS
-- [ ] Multi-application clusters
-- [ ] Partition rebalancing during broker scaling
+### Replication
+#### Scaling up
+The charm can be scaled up using `juju add-unit` command.
+```shell
+juju add-unit kafka
+```
+
+To add a specific number of brokers, an extra argument is needed:
+```shell
+juju add-unit kafka -n <num_brokers_to_add>
+```
+
+#### Scaling down
+To scale down the charm, use `juju remove-unit` command.
+```shell
+juju remove-unit <unit_name>
+```
+
+Even when scaling multiple units at the same time, the charm uses a rolling restart sequence to make sure the cluster stays available and healthy during the operation.
```

Comment:
I don't know enough about kafka 🙈 if it is advised that a user wait for all units to be idle before removing a second unit it might be worth it to add a disclaimer 😄

Reply:
The rolling restart handles this, so you can add/remove a bunch of them at the same time.